frontend/src/components/UpdateProfile/index.js

```diff
@@ -17,7 +18,14 @@ const UpdateProfile = ({
         history.push("/account/me")
```

Comment:
Do we need to do the `preexistingMedicalCondition` mapping here also for update for its own profile?

Reply:
Yes, I think we might do. Will check it out and update if needed.